templates/simple/flake.nix

```diff
@@ -3,7 +3,7 @@
   outputs = inp:
     inp.dream2nix.lib.makeFlakeOutputs {
       # modify according to your supported systems
```

Comment:
Maybe we should remove / change the comment here? "append / remove lines to the file for modifying supported systems" or something?

Reply:
I removed the comment. I think a comment is not necessary because an error will be thrown on first use explaining what to do.